README.adoc

```diff
@@ -323,41 +309,46 @@ include::finish/src/test/java/it/io/openliberty/guides/system/SystemEndpointTest
 // =================================================================================================
 
 [role='command']
-include::{common-includes}/mvnverify.adoc[]
+include::{common-includes}/devmode-test.adoc[]
+
+You will see the following output:
 
 [source, role="no_copy"]
 ----
--------------------------------------------------------
- T E S T S
--------------------------------------------------------
-Running it.io.openliberty.guides.health.HealthTest
-Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 3.504 sec - in it.io.openliberty.guides.health.HealthTest
-Running it.io.openliberty.guides.inventory.InventoryEndpointTest
-Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.326 sec - in it.io.openliberty.guides.inventory.InventoryEndpointTest
-Running it.io.openliberty.guides.system.SystemEndpointTest
-Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.011 sec - in it.io.openliberty.guides.system.SystemEndpointTest
-
-Results :
-
-Tests run: 6, Failures: 0, Errors: 0, Skipped: 0
+[INFO] -------------------------------------------------------
+[INFO]  T E S T S
+[INFO] -------------------------------------------------------
+[INFO] Running it.io.openliberty.guides.health.HealthIT
+[INFO] Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 3.463 s - in it.io.openliberty.guides.health.HealthIT
+[INFO] Running it.io.openliberty.guides.system.SystemEndpointIT
+[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.009 s - in it.io.openliberty.guides.system.SystemEndpointIT
+[INFO] Running it.io.openliberty.guides.inventory.InventoryEndpointIT
+[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.102 s - in it.io.openliberty.guides.inventory.InventoryEndpointIT
+[INFO]
+[INFO] Results:
+[INFO]
+[INFO] Tests run: 6, Failures: 0, Errors: 0, Skipped: 0
 ----
 
-To see whether the tests detect a failure, manually change the configuration of 
-[hotspot=3 file=1]`io_openliberty_guides_inventory_inMaintenance` from `false` to `true` 
-in the `resources/CustomConfigSource.json` file. Re-run the Maven build. You will see a 
-test failure occur because the initial status of the `inventory` service is `DOWN`. 
+To see whether the tests detect a failure, manually change the configuration of
+[hotspot=3 file=1]`io_openliberty_guides_inventory_inMaintenance` from `false` to `true`
+in the `resources/CustomConfigSource.json` file. Rerun the tests to see a test failure occur.
+You will see a test failure occur because the initial status of the `inventory` service is `DOWN`.
```

Comment:
Change
"You will see a test failure occur because the initial status of the `inventory` service is `DOWN`."
To
"The test failure occurs because the initial status of the `inventory` service is `DOWN`."

Reply:
Updated